DESCRIPTION

```diff
@@ -73,3 +73,13 @@ Collate:
     'importPackages.R'
     'plot.R'
 VignetteBuilder: knitr
+
+Config/reticulate:
```

Comment:
Hi, thanks for adding the python dependencies! I am just unsure if we should install via pip. I am not sure if I remember correctly, but doesnt pip install system-wide also outside of the conda environment?

Reply:
I think the "[...] PyPI with **pip** [...]" means it is installed with pip from the PyPI repos.

Reply:
yes, that's what I mean. if we set it as FALSE, the packages will be downloaded from the Anaconda repo, which has nothing to do with your concern that "doesn't  pip install system-wide also outside of the conda environment" or not. To answer your question, we may need to make sure with python/conda environment will be used by the user, I guess.